spot-client/src/common/logger/logger.js

```diff
@@ -13,35 +13,33 @@ const jitsiLogger = getLogger(null, null, { disableCallerInfo: true });
  * @returns {string} The log object, which includes metadata, as a string.
  */
 function formatMessage(level, message, context) {
-    const formattedMessage = {
-        level,
-        timestamp: Date.now(),
-        message
-    };
+    const timestamp = Date.now();
+    let contextValue;
 
     if (context) {
         if (typeof context === 'string') {
-            formattedMessage.context = context;
+            contextValue = context;
         } else {
             const contextCopy = { ...context };
 
             for (const key in contextCopy) {
-                if (contextCopy.hasOwnProperty(key)
-                    && contextCopy[key] instanceof Error) {
+                if (contextCopy.hasOwnProperty(key) && contextCopy[key] instanceof Error) {
                     const error = contextCopy[key];
 
-                    contextCopy[key] = JSON.stringify(
-                        error,
-                        Object.getOwnPropertyNames(error)
-                    );
+                    contextCopy[key] = JSON.stringify(error, Object.getOwnPropertyNames(error));
                 }
             }
-            formattedMessage.context = contextCopy;
+            contextValue = JSON.stringify(contextCopy);
         }
+    }
+
+    const finalMessage = `[level]: ${level}, [timestamp]: ${timestamp}, [message]: ${message}`;
```

Comment:
Since the logger already prints the level and timestamp, let's do it as follows: 

`message contextJSON`

Example: `Something happened { "foo": 1234 }`